### PR TITLE
Bug 1702783: Fix Bugs with Operator CR View

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Achieved by disabling virtual scrolling for this list view. Additionally, fixes incorrect redirect when deleting CR using kebab actions.

Addresses https://jira.coreos.com/browse/CONSOLE-1438
Addresses https://jira.coreos.com/browse/OLM-1056
Addresses https://bugzilla.redhat.com/show_bug.cgi?id=1702783

### CODE REVIEW

File: frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx [hunk 1]
```diff
@@ -12,10 +12,28 @@ import { SpecDescriptor } from './descriptors/spec';
...
 import { Resources } from './k8s-resource';
 import { List, MultiListPage, ListPage, ListHeader, ColHead, DetailsPage, CompactExpandButtons } from '../factory';
-import { ResourceLink, ResourceSummary, StatusBox, navFactory, Timestamp, LabelList, ResourceIcon, MsgBox, ResourceKebab, Kebab } from '../utils';
+import { ResourceLink, ResourceSummary, StatusBox, navFactory, Timestamp, LabelList, ResourceIcon, MsgBox, ResourceKebab, KebabAction } from '../utils';
 import { connectToModel } from '../../kinds';
 import { kindForReference, K8sResourceKind, OwnerReference, K8sKind, referenceFor, GroupVersionKind, referenceForModel } from '../../module/k8s';
 import { ClusterServiceVersionModel } from '../../models';
+import { deleteModal } from '../modals';
+
+const csvName = () => location.pathname.split('/').find((part, i, allParts) => allParts[i - 1] === ClusterServiceVersionModel.plural);
```
Comment (spadgett) [inline frontend/public/components/operator-lifecycle-manager/clusterserviceversion-resource.tsx]: Too bad we have to do path math, but I think it's OK.

**Comment by alecmerdler** [comment]
@spadgett thanks for changing the PR title, forgot about that!

**Pull request merged**